src/caNotification.ts

```diff
@@ -0,0 +1,48 @@
+'use strict';
+
+class CANotification {
+
+  private respData: any;
+  constructor(respData: any) {
+    this.respData = respData;
+  }
+
+  public origin(): string {
+    return this.respData.uri;
+  }
+
+  public isDone(): boolean {
+    return this.respData.done === true;
+  }
+
+  public hasWarning(): boolean {
+    return (this.respData.diagCount || 0) > 0;
+  }
+
+  public popupText(): string {
+    return this.respData.data;
+  }
+
+  private vulnCountText(): string {
+    const vulns = this.respData.vulnCount.vulnerabilityCount || 0 + this.respData.vulnCount.advisoryCount || 0;
```

Comment:
const vulns = (this.respData.vulnCount.vulnerabilityCount || 0) + (this.respData.vulnCount.advisoryCount || 0);

because the existing is considering something like this: 
const vulns = (this.respData.vulnCount.vulnerabilityCount) || (0 + this.respData.vulnCount.advisoryCount) || (0);

Reply:
@dvandra , Fixed it. Please check now.